string segment_id
---
bool result
soma_llsd_msgs/Segment response
